hyperspy/_signals/eds.py

```diff
@@ -926,9 +927,15 @@ def plot(self,
         set_elements, add_elements, estimate_integration_windows,
         get_lines_intensity, estimate_background_windows
         """
+        if display_range is None:
+            axis = self.axes_manager[-1].axis
+            if len(axis) > 1 and axis[0] < 0.1:
```

Comment:
This assumes that the units are keV.

By the way, is this documented?

Reply:
Yes, you are right!